apps/web/src/pages/quick-start/components/demo-flow/nodes/NodeButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { UnstyledButton } from '@mantine/core';
+
+import { useStyles } from '../../../../../design-system/template-button/TemplateButton.styles';
+import { colors, Text } from '../../../../../design-system';
+
+export function NodeButton({
+  data,
+  id,
+  Handlers,
+  Icon,
+  ActionItem,
+}: {
+  data: any;
+  id: string;
+  Handlers: React.FC<any>;
+  Icon: React.FC<any>;
+  ActionItem?: React.FC<any>;
+}) {
+  const { theme } = useStyles();
+
+  return (
+    <>
+      <Button
+        data-test-id={`data-test-id-${data.label}`}
+        sx={{
+          backgroundColor: theme.colorScheme === 'dark' ? colors.B17 : colors.white,
+        }}
+      >
```

Comment:
I'm just wondering why this one is defined as the button? I mean I don't see the need and we don't attach any click listeners... Did I miss something?

Reply:
In the start I wanted to get a button design but looks like In the end design was changed, I will review it again, thanks for raising it up!! :)

Reply:
Addressed it on the following PR
https://github.com/novuhq/novu/pull/2968